classes/musicAlbum.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'item'
+
+# MusicAlbum.
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  @music_albums = []
+  def initialize(on_spotify:, **attributes)
+    super(attributes)
```

Comment:
Since we have to pass the publish_date parameter to super  Items class change attribute with publish_date